dash-spv-bench/run.sh

```
@@ -0,0 +1,329 @@
+      || cp -R "${CHAIN_DIR}" "${dst}"
+  done
+
+  local batch=4
+  echo "==> starting ${n} peers in batches of ${batch}"
+  local started="" count=0
+  for svc in ${services}; do
+    CLONE_DIR="${clone_dir}" compose up -d "${svc}" >/dev/null 2>&1
+    started="${started} spv-bench-${svc}"; count=$((count + 1))
+    if [ $((count % batch)) -eq 0 ]; then
+      echo -n "  loaded ${count}/${n} "
+      wait_loaded ${started} || { echo " timeout loading batch" >&2; exit 1; }
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)